client/components/banner.jsx

```diff
@@ -1,40 +1,12 @@
 import React from 'react';
-import style from '../bootstrap.min.css';
+import style from '../trail-style.css';
 
 export default class Banner extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
-      trail: {
-        type: 'trail',
-        id: '1',
-        attributes: {
-          trailName: 'Golden Gate Park Trail',
-          distance: 6.1,
-          distanceUnits: 'miles',
-          elevationGain: 351,
-          elevationUnits: 'ft',
-          description: 'Golden Gate Park Trail is a 6.1 mile heavily trafficked loop trail located near San Francisco, California that features a lake and is good for all skill levels. The trail offers a number of activity options and is accessible year-round. Dogs are also able to use this trail but must be kept on leash.',
-          routeType: 'Loop',
-          difficulty: 'Easy',
-          tags: [
-            'dogs on leash',
-            'kid friendly',
-            'birding',
-            'hiking',
-            'mountain biking',
-            'nature trips',
-            'road biking',
-            'trail running',
-            'walking',
-            'forest',
-            'lake',
-            'partially paved'
-          ],
-          'general_area': 'Golden Gate Park',
-          'origin': 'https://www.alltrails.com/trail/us/california/golden-gate-park-trail'
-        }
-      },
+      fetching: true,
+      trail: {},
       heroPhoto: {
```

Comment:
eventually this will be get from Meng's service right?

Reply:
css is [wip], but since you mention it, doesn't match what?

And, yes heroPhoto data will come from Meng's service, working on that in my current feature branch.